perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -188,7 +187,8 @@ sub problem_as_hashref {
             # no banner_id as hidden
         } elsif ( $problem->state eq 'investigating' ) {
             $hashref->{state} = 'wish'; # is this correct?
```

Comment:
Is it correct?

Reply:
Who knows, it's been so long, let's assume it was? ;) I'm going to delete all this when moving to states in the DB, so hooray!